components/layout_2020/query.rs

```diff
@@ -531,6 +514,29 @@ fn process_offset_parent_query_inner(
     })
 }
 
+/// Returns whether or not the element with the given style and body element determination
+/// is eligible to be a parent element for offset* queries.
```

Comment:
This makes it sound like it's also for `offsetLeft` and such, but it's only for `offsetParent`, isn't it?

Reply:
I think this is used for `offsetLeft` and friends, because they depend on the definition of `offsetParent`. For instance, the definition of `OffsetLeft()` in `htmlelement.rs` is:

```rust
    // https://drafts.csswg.org/cssom-view/#dom-htmlelement-offsetleft
    fn OffsetLeft(&self) -> i32 {
        if self.is::<HTMLBodyElement>() {
            return 0;
        }

        let node = self.upcast::<Node>();
        let window = window_from_node(self);
        let (_, rect) = window.offset_parent_query(node);

        rect.origin.x.to_nearest_px()
    }
```

Reply:
Sorry you are correct, I think I tried the other day and it didn't seem to work, but I probably made a mistake.